tests/testsuite/cargo_information/pick_msrv_compatible_package/mod.rs

```diff
@@ -0,0 +1,23 @@
+use cargo_test_macro::cargo_test;
+use cargo_test_support::curr_dir;
+
+use super::{cargo_info, init_registry_without_token};
+
+#[cargo_test]
+fn case() {
+    init_registry_without_token();
+    cargo_test_support::registry::Package::new("my-package", "0.1.1+my-package")
+        .rust_version("1.0.0")
+        .publish();
+    cargo_test_support::registry::Package::new("my-package", "0.2.0+my-package")
+        .rust_version("1.90.0")
```

Comment:
If you are testing against `rustc --version`, then this test should use a ridiculously high version.  For example, one cargo test uses `rust-version = "1.9876.0"`

Reply:
Updated. Thanks!